packages/gitgraph-core/src/template.ts

```diff
@@ -0,0 +1,400 @@
+import * as merge from "lodash.merge";
```

Comment:
This fails on Typescript 2.8.3 for me (after I manually installed `@types/lodash.merge`

```
src/template.ts:1:24 - error TS2497: Module '"/path/to/gitgraph.js/packages/gitgraph-core/node_modules/@types/lodash.merge/index"' resolves to a non-module entity and cannot be imported using this construct.
```